_includes/homepage_top.html

```diff
@@ -28,6 +28,8 @@
     </div>
 
     <div class="gap">
-        <a href ="https://github.com/raml-org/raml-spec/blob/master/versions/raml-10/raml-10.md/" target="_blank" class="raml-links">Read the Specification</a>
+        <a href ="https://github.com/raml-org/raml-spec/blob/master/versions/raml-10/raml-10.md/" target="_blank" class="raml-links">RAML 1.0 Specification</a><br />
```

Comment:
I think stating the version number is nice; otherwise we are asking the user to do work to figure it out. And it it not that much trouble to update. But it is your call.